### ISSUE TITLE
Incorrect use of `interpret_d`

### ISSUE THREAD

**Issue by mattansb** [easystats/report#204]
Standardized slopes are not standardized differences!
https://github.com/easystats/report/blob/a59f6b3caa759e461ad8358b56e3e26dd1ef3e4d/R/report.lm.R#L82

There is *no* simple 1:1 standardized interpretation of standardized slopes in multiple regression.

**Comment by mattansb** [comment]
Best you can do here is verbal interpretation ("A change of 1 SD in X corresponds to a [conditional] change of {} units in Y....").

**Comment by bwiernik** [comment]
It's the intercept though, so this is the standardized Y when all X's are at mean. I don't really see the value of interpreting the intercept for a standardized normal response. But if you do, d-like benchmarks seem reasonable? How many SDs from zero is y at the mean of X's?

**Comment by mattansb** [comment]
> It's the intercept though [...]

Where are you seeing that?

**Comment by bwiernik** [comment]
Oh, apparently nowhere but in my "just woke up" fantasy land 😅

**Comment by bwiernik** [comment]
My general thinking here is that, multicollinearity issues aside, standardized coefficients are generally similar to either partial correlations (continuous X) or partial standardized mean differences (factor X). Applying those interpretation rules is generally reasonable, perhaps with a caveat if VIF is high?

**Comment by mattansb** [comment]
Are there such benchmarks for interpretation of partial correlations / partial standardized mean differences?

If so, I know at least one Frenchman who will be very happy to see this happen!

**Comment by bwiernik** [comment]
Not really. My recommendation would be to use the zero order r/d benchmarks and add a note about them maybe not applying to partial effect sizes?

**Comment by mattansb** [comment]
Ew :(

**Comment by bwiernik** [comment]
Not sure the opposition? These are all super rough guidelines in any case, and generally we might expect the partial effects to be "better" estimates of the true effect assuming our covariates are carefully chosen

**Comment by mattansb** [comment]
> [...] assuming our covariates are carefully chosen

That's quite an assumption. 

Even a VIF of 4.9 ("low") is an R<sup>2</sup><sub>X</sub> of 0.8... Which is *not* small, and can still hinders interpretation....

**Comment by mattansb** [comment]
Anyway, just added `interpret_vif` (:

**Comment by bwiernik** [comment]
> That's quite an assumption.

We can't automate thinking, just provide encouragement toward better approaches 😀

**Comment by mattansb** [comment]
Fine.... I concede!
@DominiqueMakowski I think you owe @bwiernik a beer (:
